server/routerlicious/packages/tinylicious/src/services/webServerFactory.ts

```diff
@@ -71,7 +71,7 @@ class SocketIoServer extends EventEmitter implements IWebSocketServer {
 	}
 
 	public async close(): Promise<void> {
-		await new Promise<void>((resolve) => this.io.close(() => resolve()));
+		await this.io.close();
```

Comment:
Necessary due to breaking change in socket.io `4.7.5->4.8.0`.

Reply:
Can we get even shorter? If `this.io.close()` returns `Promise<void>`, then this function doesn't even need to be async anymore, correct? You can just do:

```ts
public close(): Promise<void> {
  return this.io.close();
}
```

Reply:
Yep, updated.